src/app/loadout-builder/filter/LoadoutOptimizerExotic.tsx

```diff
@@ -0,0 +1,163 @@
+import ClosableContainer from 'app/dim-ui/ClosableContainer';
+import { t } from 'app/i18next-t';
+import { allItemsSelector, createItemContextSelector } from 'app/inventory/selectors';
+import { makeFakeItem } from 'app/inventory/store/d2-item-factory';
+import LoadoutEditSection from 'app/loadout/loadout-edit/LoadoutEditSection';
+import { useD2Definitions } from 'app/manifest/selectors';
+import { DestinyClass } from 'bungie-api-ts/destiny2';
+import anyExoticIcon from 'images/anyExotic.svg';
+import noExoticIcon from 'images/noExotic.svg';
+import noExoticPreferenceIcon from 'images/noExoticPreference.svg';
+import _ from 'lodash';
+import { Dispatch, memo, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { LoadoutBuilderAction } from '../loadout-builder-reducer';
+import { LOCKED_EXOTIC_ANY_EXOTIC, LOCKED_EXOTIC_NO_EXOTIC } from '../types';
+import ExoticPicker, { findLockableExotics, resolveExoticInfo } from './ExoticPicker';
+import { exoticTileInfo } from './ExoticTile';
+import styles from './LoadoutOptimizerExotic.m.scss';
+
+const LoadoutOptimizerExotic = memo(function LoadoutOptimizerExotic({
+  classType,
+  className,
+  storeId,
+  lockedExoticHash,
+  lbDispatch,
+}: {
+  classType: DestinyClass;
+  storeId: string;
+  className?: string;
+  lockedExoticHash: number | undefined;
+  lbDispatch: Dispatch<LoadoutBuilderAction>;
+}) {
+  const [showExoticPicker, setShowExoticPicker] = useState(false);
+  const defs = useD2Definitions()!;
+  const allItems = useSelector(allItemsSelector);
+
+  const handleClear = () => {
+    lbDispatch({ type: 'removeLockedExotic' });
+  };
+
+  const handleSyncFromEquipped = () => {
+    const equippedExotic = allItems.find(
+      (i) => i.equipped && i.isExotic && i.bucket.inArmor && i.owner === storeId,
```

Comment:
This could theoretically be an equipped Armor 1.0/1.5 exotic which is not available in Loadout Optimizer but that's pretty unlikely to happen.